src/sonic-host-service/org.sonic.hostservice.conf

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE busconfig PUBLIC
+ "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
+<busconfig>
+
+  <!-- Only root can own the bus -->
+
+  <policy user="root">
+    <allow own_prefix="org.SONiC.HostService"/>
```

Comment:
The owner is restricted to root, since that's the user as which the host service runs. However, the policy on the line below controls which users can actually read/write to the bus.

Reply:
Every docker runs as root. So this does not help. :-)

Would like to confirm, if container level restriction is possible through .conf settings or not.

Reply:
We can restrict on a user/group basis, not on container basis. If we can adjust the REST server or telemetry server to run as non-root user, then we can restrict the access further.

Reply:
That is fine. As long as we don't mount /var/run/dbus are rw, to other containers, it is safe.